test/jira-tests.js

```diff
@@ -539,9 +539,45 @@ describe('Jira API Tests', () => {
       result.should.eql('http://jira.somehost.com:8080/rest/api/2.0/component/someComponentNumber');
     });
 
-    it('listFields hits proper url', async () => {
```

Comment:
No, I didn't remove it. I grouped it under "Field APIs Suite Tests" along with createCustomFields as described in [Jira standard ](https://docs.atlassian.com/jira/REST/cloud/#api/2/field)